packages/quiz/package.json

```diff
@@ -11,8 +11,10 @@
   ],
   "scripts": {
     "test": "jest",
-    "dev": "tsc -w",
-    "build": "tsc",
+    "dev": "tsc -w & pnpm watch:css",
+    "watch:css": "node-sass --output-style expanded --source-map true --source-map-contents true --precision 6 src/styles.css dist/styles.css -w",
```

Comment:
could probably be `pnpm build:css -- -w`

Reply:
nice — updated 👍